condominium/data/ir_actions_server.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="ir_act_server_split_per_property" model="ir.actions.server">
+        <field name="name">Split per Property</field>
+        <field name="state">code</field>
+        <field name="model_id" ref="sale.model_sale_order"/>
+        <field name="code"><![CDATA[
+for property in record.partner_id.x_condominium_units_ids:
+    record['state'] = 'cancel'
+    new_so = record.copy()
+    new_so['x_source_sales_order'] = record.id
+    new_so['partner_id'] = property.x_partner_id.id
+    new_so['x_property_id'] = property
+for sol in new_so.order_line:
+    sol['price_unit'] = sol.price_unit * property.x_area_percent
+]]></field>
+    </record>
+    <record id="ir_act_server_749" model="ir.actions.server">
+        <field name="name">Confirm</field>
+        <field name="state">code</field>
+        <field name="model_id" ref="sale.model_sale_order"/>
+        <field name="binding_model_id" ref="sale.model_sale_order"/>
+        <field name="code"><![CDATA[
+for so in records:
+    so.action_confirm()
```

Comment:
Reading twice... Why do we need this server action? Couldn't we call action_confirm directly on the view?

Also, no need to loop as this method works on a recordset.
```suggestion
records.action_confirm()
```